Gigs/Gigs/Application/GigController.swift

```diff
@@ -168,18 +168,18 @@ class GigController {
         var request = URLRequest(url: newGigsUrl)
         request.httpMethod = HTTPMethod.post.rawValue
         request.setValue("Bearer \(bearer.token)", forHTTPHeaderField: "Authorization")
+        request.setValue("application/json", forHTTPHeaderField: "Content-Type")
```

Comment:
You need to be sure you include this whenever you are going to be posting anything. This let's the api know that you are going to be sending it some json content